bundle/Command/DeleteOldCollectedInfoCommand.php

```diff
@@ -0,0 +1,206 @@
+    {
+        $this->infoCollection = $infoCollection;
+        // Parent constructor call is mandatory for commands registered as services
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('nginfocollector:delete');
+        $this->setDescription('Deletes collected data that is older than specified period.');
+        $this->setHelp('This command allows you to delete data collected by this library that is older than specified period.');
+
+        $this->setDefinition(
+            new InputDefinition(
+                [
+                    new InputOption('content-id', 'c', InputOption::VALUE_REQUIRED, 'Content id.'),
+                    new InputOption('field-identifiers', 'f', InputOption::VALUE_REQUIRED, 'Field definition identifiers list.'),
+                    new InputOption('period', 'p', InputOption::VALUE_REQUIRED, 'Attributes older that this period will be deleted.'),
+                    new InputOption('all', 'a', InputOption::VALUE_NONE, 'Delete all fields.'),
+                    new InputOption('neglect', 'nn', InputOption::VALUE_NONE, 'Do not ask for confirmation.'),
+                ],
+            ),
+        );
+
+        $this->addUsage('--content-id=123 --field-identifiers=title,name,last_name');
+        $this->addUsage('--info-collection-id=456 --field-identifiers=title,name,last_name');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (null === $input->getOption('content-id')) {
+            $output->writeln('<error>                                       </error>');
+            $output->writeln('<error>     Missing content-id parameter.     </error>');
+            $output->writeln('<error>                                       </error>');
+
+            return $this->displayHelp($input, $output);
+        }
+
+        if (null === $input->getOption('field-identifiers') && !$input->getOption('all')) {
+            $output->writeln('<error>                                              </error>');
+            $output->writeln('<error>     Missing field-identifiers parameter.     </error>');
+            $output->writeln('<error>                                              </error>');
+
+            return $this->displayHelp($input, $output);
+        }
+
+        if (null === $input->getOption('period')) {
+            $output->writeln('<error>                                       </error>');
+            $output->writeln('<error>     Missing period parameter.         </error>');
+            $output->writeln('<error>                                       </error>');
+
+            return $this->displayHelp($input, $output);
+        }
+
+        $contentId = (int) $input->getOption('content-id');
+        $fields = $this->getFields($input);
+
+        $info = sprintf('Command will delete <info>%s</info> fields for content #%d', empty($fields) ? 'all' : implode(', ', $fields), $contentId);
```

Comment:
Do not use `empty`, but `mb_strlen($fields) === 0`

Reply:
Fixed